vertical-pod-autoscaler/pkg/recommender/logic/recommender.go

```diff
@@ -37,6 +37,11 @@ var (
 	humanizeMemory             = flag.Bool("humanize-memory", false, "Convert memory values in recommendations to the highest appropriate SI unit with up to 2 decimal places for better readability.")
 )
 
+// GetHumanizeMemory returns the value of the HumanizeMemory flag.
+func GetHumanizeMemory() bool {
+	return *humanizeMemory
+}
```

Comment:
I'm curious what the purpose of this change is?

Reply:
I needed that for having accessible in https://github.com/kubernetes/autoscaler/pull/7550/files#diff-27dae50f34ff6f13aa0bcf2556de814c2d2e7aa2dbc277e1ee8339f60917622bR227-R230.